modules/onboarding-simulation/application/services/generate-concierge-turn.ts

```
@@ -0,0 +1,181 @@
+	};
+}
+
+function createSimulationTool(
+	deps: ConciergeGenerationDependencies,
+	params: ConciergeGenerationParams,
+	ctx: ToolContext,
+) {
+	return tool({
+		description: COMPLETE_ONBOARDING_TOOL_DESCRIPTION,
+		inputSchema: completeOnboardingInputSchema,
+		execute: async (input) => {
```

Biome (2.4.4)

[error] Excessive complexity of 14 detected (max: 12).

(lint/complexity/noExcessiveCognitiveComplexity)